### ISSUE TITLE
Support purs v0.13.3

### ISSUE THREAD

**Issue by kurtmilam** [ajnsit/purescript-concur-starter#6]
I'm running `purs v0.13.3` and `spago v0.9.0.0`.

I cloned the repo, removed the dev dependencies on `purescript` and `spago`, ran `spago upgrade-set`, then followed the installation instructions and saw these errors (while executing `npm run dev`):

```
Installation complete.
[1/5 ModuleNotFound] src/Main.purs:5:1

  5  import Concur.Core (Widget)
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^

  Module Concur.Core was not found.
  Make sure the source file exists, and that it has been provided as an input to the compiler.

[2/5 ModuleNotFound] src/Main.purs:6:1

  6  import Concur.React (HTML)
     ^^^^^^^^^^^^^^^^^^^^^^^^^^

  Module Concur.React was not found.
  Make sure the source file exists, and that it has been provided as an input to the compiler.

[3/5 ModuleNotFound] src/Main.purs:7:1

  7  import Concur.React.DOM as D
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  Module Concur.React.DOM was not found.
  Make sure the source file exists, and that it has been provided as an input to the compiler.

[4/5 ModuleNotFound] src/Main.purs:8:1

  8  import Concur.React.Props as P
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  Module Concur.React.Props was not found.
  Make sure the source file exists, and that it has been provided as an input to the compiler.

[5/5 ModuleNotFound] src/Main.purs:9:1

  9  import Concur.React.Run (runWidgetInDom)
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  Module Concur.React.Run was not found.
  Make sure the source file exists, and that it has been provided as an input to the compiler.
```

I had tried a similar procedure a ~day earlier with more success (the project builds and runs). On this attempt, I cloned the repo, then followed the installation instructions. I then removed the dev dependencies on purescript and spago from package.json, ran `spago upgrade-set`, then `npm run dev`, and everything worked.

**Comment by kurtmilam** [comment]
I noticed that the version of `concur-react` was `v0.3.8` in the working attempt, while it was `v0.3.9` in the failed attempt.

I didn't expect this (i.e. removing the dev dependency on `purescript v0.12.3` and running `pago upgrade-set`) to work, but was pleasantly surprised when it did on the initial attempt, then confused when it didn't work on the second installation, which I performed shortly after the first (within 24 hours, maybe significantly less).
